### PR TITLE
Add _Runtime and Reflection to Windows installer

### PR DESCRIPTION
**Pull request by Azoy**

### CODE REVIEW

File: platforms/Windows/sdk-amd64.wxs [hunk 1]
```diff
@@ -497,6 +533,8 @@
       <ComponentGroupRef Id="dispatch" />
       <ComponentGroupRef Id="BlocksRuntime" />
       <ComponentGroupRef Id="SwiftRemoteMirror" />
+      <ComponentGroupRef Id="Reflection" />
+      <ComponentGroupRef Id="_Runtime" />
```
Comment (compnerd) [inline platforms/Windows/sdk-amd64.wxs:537]: Would be nice to keep this list sorted.

File: platforms/Windows/sdk-arm64.wxs [hunk 2]
```diff
@@ -497,6 +533,8 @@
       <ComponentGroupRef Id="dispatch" />
       <ComponentGroupRef Id="BlocksRuntime" />
       <ComponentGroupRef Id="SwiftRemoteMirror" />
+      <ComponentGroupRef Id="Reflection" />
+      <ComponentGroupRef Id="_Runtime" />
```
Comment (compnerd) [inline platforms/Windows/sdk-arm64.wxs:537]: Similar

File: platforms/Windows/sdk-x86.wxs [hunk 3]
```diff
@@ -240,6 +244,38 @@
...
+      </Component>
+      <Component Id="_Runtime.swiftmodule" Directory="_Runtime.swiftmodule" Guid="f9f10cf0-d59b-4f60-9efc-a17226f5f590">
+        <File Id="_Runtime.swiftmodule" Source="$(var.SDK_ROOT)\usr\lib\swift\windows\_Runtime.swiftmodule\i686-unknown-windows-msvc.swiftmodule" Checksum="yes" />
+      </Component>
+
+      <Component Id="swift_Runtime.lib" Directory="WindowsSDK_usr_lib_swift_windows_i686" Guid="f4462968-709d-44bc-8657-16e654946e3e">
+        <File Id="swift_Runtime.lib" Source="$(var.SDK_ROOT)\usr\lib\swift\windows\i686\swift_Runtime.lib" Checksum="yes" />
+      </Component>
+    </ComponentGroup>
+
```
Comment (compnerd) [inline platforms/Windows/sdk-x86.wxs:278]: Missing update for the component list?

**Review by compnerd — APPROVED** [review]
LGTM assuming that the cross-repo test passes!

**Comment by Azoy** [comment]
Tests have passed, merging.

**Pull request merged**